dotCMS/src/main/java/com/dotcms/rest/api/v1/taillog/TailLogResource.java

```diff
@@ -0,0 +1,214 @@
+package com.dotcms.rest.api.v1.taillog;
+
+import static com.dotcms.util.CollectionsUtils.map;
+
+import com.dotcms.repackage.org.apache.commons.io.input.TailerListenerAdapter;
```

Comment:
Maybe try to use the non-repackaged version of this class?

Reply:
Done